### DISCUSSION TITLE
Metadata Backup

### DISCUSSION THREAD

**Discussion by timiil** [directus/directus#7346] [Feature Requests]
Hi , do we have any plan to export / import the Metadata in Admin pannel ? or just expose some API to let us import the 'directus_collections', 'directus_fields', ... searvel collections ?

**Comment by rijkvanzanten** [comment]
See https://github.com/directus/directus/discussions/3891

**Comment by br41nslug** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
